`PyaroConfig` from `pyaerocom.io.pyaro.pyaro_config` is too pedantic

**Is your feature request related to a problem? Please describe.**
`PyaroConfig` from `pyaerocom.io.pyaro.pyaro_config` is too pedantic.
Example config:
```python
obsconfig = PyaroConfig(
            name=data_name,
            data_id=data_id,
            filename_or_obj_or_url=url,
            filters=station_filter,)
```
`filename_or_obj_or_url` can only be a string but not list of strings. It should also allow a list of strings and maybe also a list of path like objects (both!; not only path like objects since there are pyaro readers that use this differently)

**Describe the solution you would like to see**
allow a list of strings and a list of path like objects in addition

**Additional context**
This error comes up when supplying a list:
```
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for PyaroConfig
E       filename_or_obj_or_url
E         Input should be a valid string [type=string_type, input_value=['concso4t', 'concso4c'], input_type=list]
E           For further information visit https://errors.pydantic.dev/2.9/v/string_type

test_PyerocomBinding.py:91: ValidationError
```

--- a/pyaerocom/io/pyaro/pyaro_config.py
+++ b/pyaerocom/io/pyaro/pyaro_config.py
@@ -3,7 +3,7 @@ from __future__ import annotations
 import logging
 from importlib import resources
 from pathlib import Path
-from typing import ClassVar, Union
+from typing import ClassVar
 
 import yaml
 from pydantic import BaseModel, ConfigDict
@@ -29,7 +29,7 @@ class PyaroConfig(BaseModel):
 
     name: str
     data_id: str
-    filename_or_obj_or_url: Union[str, list[str], Path, list[Path]]
+    filename_or_obj_or_url: str | list[str] | Path | list[Path]
     filters: dict[str, dict[str, list[str]] | dict[str, list[tuple]]]
     name_map: dict[str, str] | None = None  # no Unit conversion option
 

--- a/pyaerocom/io/pyaro/pyaro_config.py
+++ b/pyaerocom/io/pyaro/pyaro_config.py
@@ -3,7 +3,7 @@ from __future__ import annotations
 import logging
 from importlib import resources
 from pathlib import Path
-from typing import ClassVar
+from typing import ClassVar, Union
 
 import yaml
 from pydantic import BaseModel, ConfigDict
@@ -28,7 +28,7 @@ class PyaroConfig(BaseModel):
 
     name: str
     data_id: str
-    filename_or_obj_or_url: [str, list[str], Path, list[Path]]
+    filename_or_obj_or_url: Union [str, list[str], Path, list[Path]]
     filters: dict[str, dict[str, list[str]] | dict[str, list[tuple]]]
     name_map: dict[str, str] | None = None  # no Unit conversion option
 

--- a/pyaerocom/io/pyaro/pyaro_config.py
+++ b/pyaerocom/io/pyaro/pyaro_config.py
@@ -12,6 +12,7 @@ import pyaerocom as pya
 
 logger = logging.getLogger(__name__)
 
+
 # TODO Check a validator if extra/kwarg is serializable. Either in json_repr or as a @field_validator on extra
 
 
@@ -28,7 +29,7 @@ class PyaroConfig(BaseModel):
 
     name: str
     data_id: str
-    filename_or_obj_or_url: Union [str, list[str], Path, list[Path]]
+    filename_or_obj_or_url: Union[str, list[str], Path, list[Path]]
     filters: dict[str, dict[str, list[str]] | dict[str, list[tuple]]]
     name_map: dict[str, str] | None = None  # no Unit conversion option
 

--- a/pyaerocom/io/pyaro/pyaro_config.py
+++ b/pyaerocom/io/pyaro/pyaro_config.py
@@ -28,7 +28,7 @@ class PyaroConfig(BaseModel):
 
     name: str
     data_id: str
-    filename_or_obj_or_url: str
+    filename_or_obj_or_url: [str, list[str], Path, list[Path]]
     filters: dict[str, dict[str, list[str]] | dict[str, list[tuple]]]
     name_map: dict[str, str] | None = None  # no Unit conversion option
 

